src/App.js

```diff
@@ -20,54 +21,59 @@ import {
 } from './pages';
 
 function App() {
+
+  const { isSidebarOpen } = useProductsContext();
+  const overflowPropertyToHideScroll=isSidebarOpen===true?'hidden':'scroll'
```

Comment:
Semicolons missing (please follow the existing code style)

Also, add one blank line after this